src/platform/graphics/graphics-device.js

```diff
@@ -661,6 +661,14 @@ class GraphicsDevice extends EventHandler {
         }
     }
 
+    /**
+     * Clears the vertex buffer set on the graphics device. This is called automatically by the
+     * renderer.
+     */
```

Comment:
it needs `@ignore` tag to not make it a public API

Reply:
Oh, missed that, I'll take care of it.